src/main/java/world/bentobox/level/commands/AdminTop.java

```diff
@@ -45,8 +45,13 @@ public boolean execute(User user, String label, List<String> args) {
             Island island = getPlugin().getIslands().getIsland(world, topTen.getKey());
             if (island != null) {
                 rank++;
-                String item = String.valueOf(rank) + ":" + island.getName() + " "
-                        + user.getTranslation("topten.islandLevel", "[level]", String.valueOf(topTen.getValue()));
+                String item = user.getTranslation("admin.topten",
+                    "[rank]",
+                    "[name]",
+                    "[level]",
+                    String.valueOf(rank),
+                    this.getPlugin().getPlayers().getUser(island.getOwner()).getName(),
+                    String.valueOf(topTen.getValue()));
```

Comment:
That is not how the text variables are working. They are in pairs, like this: `"variable", "replacement"`.
In the current state, `[rank]` would be replaced by `[name]`, `[level]` by the rank, etc...